core/amber/src/main/resources/application.conf

```diff
@@ -5,3 +5,28 @@ constants {
     data-volume-per-node = 10
 }
 
+kamon {
```

Comment:
Try to put this config in a separate config file. And please update the indentation to match other configs. 

For the meaning of those config entries, can you add comments or references so we understand what they are?